contracts/ClaimsManager.sol

```diff
@@ -324,18 +316,23 @@ contract ClaimsManager is
             block.timestamp <= policyState.claimsAllowedUntil,
             "Claims not allowed anymore"
         );
-        claimIndex = ++claimCount;
-        claims[claimIndex] = Claim({
-            policyHash: policyHash,
+        claimHash = keccak256(
+            abi.encodePacked(
+                policyHash,
+                msg.sender,
+                beneficiary,
+                claimAmountInUsd,
+                evidence
+            )
+        );
```

Comment:
I don't think anything prevents the user from calling this function with identical params as an existing claim, so we should probably validate that we don't have an existing claim for the derived `claimHash`. Otherwise we would override the existing claim's state?

Reply:
Good point, I added that check in https://github.com/api3dao/claims-manager/pull/50/commits/a4a9f593e056d7297e55a73d963c0ef535fb534f
`evidence` will be unique to the claim, which means we can expect unique hashes per claim